code/modules/research/nanites/nanite_programs/utility.dm

```diff
@@ -258,6 +258,32 @@
 			SEND_SIGNAL(infectee, COMSIG_NANITE_SYNC, nanites)
 			infectee.investigate_log("was infected by spreading nanites by [key_name(host_mob)] at [AREACOORD(infectee)].", INVESTIGATE_NANITES)
 
+/datum/nanite_program/triggered/nanite_sting
+	name = "Nanite Sting"
+	desc = "Stings a random non-host around the host with a barely-visible cluster of nanites, making them a new host. The sting might be felt."
+	trigger_cost = 5
+	trigger_cooldown = 30
+	rogue_types = list(/datum/nanite_program/glitch, /datum/nanite_program/toxic)
+
+/datum/nanite_program/triggered/nanite_sting/trigger()
+	if(!..())
+		return
+	var/list/mob/living/target_hosts = list()
+	for(var/mob/living/L in oview(1, host_mob))
+		if(!(L.mob_biotypes & (MOB_ORGANIC|MOB_UNDEAD)))
+			continue
+		target_hosts += L
+		if(!target_hosts.len)
```

Comment:
This will never run.

Reply:
you add something to the list before this one

Reply:
nevermind you fucked up indentation, this is supposed to run outside of the for loop. Rifght now you run all of the code every time you loop through, which means you infect multiple people.